src/gtl/components/pagination/paginationComponent.ts

```diff
@@ -5,10 +5,20 @@
  * @name PaginationController
  */
 export class PaginationController {
+  public isChecked: boolean = false;
+  public settings: any;
+
   public onSelectAll: (args: {isSelected: boolean}) => void;
   public onChangeSort: (args: {sortId: number, isAscending: boolean}) => void;
   public onChangePage: (args: {pageNumber: number}) => void;
   public onChangePerPage: (args: {item: number}) => void;
+
+
```

Comment:
ERROR in ./src/gtl/components/pagination/paginationComponent.ts
[16, 1]: Consecutive blank lines are forbidden
```
 @ ./src/gtl/components/pagination/index.ts 3:28-60
 @ ./src/gtl/components/index.ts
 @ ./src/gtl/index.ts
```